src/code/operation.js

```diff
@@ -1287,6 +1287,21 @@ export default class WasabeeOp {
     for (const b of op.blockers) this.blockers.push(b); // do not use addBlocker
   }
 
+  mergeZones(op) {
```

Comment:
does this account for zone deletion?

Reply:
does it need to?

Reply:
no, it's more an "import" than a "merge"
it's there mainly for the team import